malasakit-django/pcari/models.py

```diff
@@ -106,7 +106,14 @@ def get_queryset(self):
         return queryset
 
 
-class Response(models.Model):
+class ViewPermissionMixin(models.Model):
```

Comment:
This model is unnecessary, since it isn't concrete and does not pass any fields to children. The only reason this class exists is to pass `default_permissions` to each `Meta`, in which case we should have just a `Meta` parent class.

Reply:
Thanks for pointing this out. Im going to replace it with a Meta parent.